DotnetCLIVersion.netcoreapp.txt

```diff
@@ -0,0 +1 @@
+3.0.100-alpha1-009622
```

Comment:
Should this file suffix be netcoreapp3.0?

Reply:
Or netcoreapp30

Reply:
the file will be used for future netcorapp versions too like 3.1 or 4

Reply:
what about, netcoreapp.latest.txt ?

Reply:
that sounds more appropriate. thanks I will change that